pom.xml

```diff
@@ -261,9 +263,13 @@
           <plugin>
             <groupId>org.jacoco</groupId>
             <artifactId>jacoco-maven-plugin</artifactId>
-            <version>0.8.1</version>
             <configuration>
               <output>file</output>
+              <excludes>
+                <exclude>**/*.sketch</exclude>
```

Comment:
@olenagerasimova why exclude this? there are no `.sketch` file in this repository, is there?